src/helpers/hashPassword.js

```diff
@@ -0,0 +1,5 @@
+import bcrypt from 'bcrypt';
```

Comment:
Since bcrypt is used for hashing and verifying passwords, put it in a service folder and name it `bcrypt.js`